nodeup/pkg/model/kops_controller.go

```diff
@@ -62,6 +62,9 @@ func (b *KopsControllerBuilder) Build(c *fi.ModelBuilderContext) error {
 		Subject:        nodetasks.PKIXName{CommonName: "kops-controller"},
 		AlternateNames: []string{"kops-controller.internal." + b.Cluster.ObjectMeta.Name},
 	}
+	if b.BootConfig.APIServer != "" {
+		issueCert.AlternateNames = append(issueCert.AlternateNames, b.BootConfig.APIServer)
```

Comment:
kops-controller isn't the apiserver, so shouldn't have the apiserver domain in its SANs.

Reply:
This is only set when using `--dns=none` and represents the IP address of the API server.

Reply:
Perhaps rename it to `APIServerIP`? And what about HA control planes? Or IPv6 clusters, where APIServer nodes are dual-stack?

Reply:
Or is it `APIServerLoadBalancerIP`?

Reply:
`APIServerIP` it is :).